code401challenges/src/test/java/code401challenges/linkedList/LinkedListTest.java

```diff
@@ -73,4 +73,49 @@
         assertEquals(expectedSecondFromEnd, pendingSecondFromEnd);
     }
 
+    @Test public void testMergeEqualLengthLists(){
+        LinkedList actual1 = new LinkedList();
+        actual1.insert(1);
+        actual1.insert(2);
+        actual1.insert(3);
+
+        LinkedList actual2 = new LinkedList();
+        actual2.insert(5);
+        actual2.insert(9);
+        actual2.insert(4);
+
+        LinkedList actual = LinkedList.mergeLists(actual1, actual2);
+        String expected = "{3} -> {4} -> {2} -> {9} -> {1} -> {5} -> NULL";
+        assertEquals(expected, actual.toString());
+    }
+
+    @Test public void testMergeUnevenList(){
+        LinkedList actual1 = new LinkedList();
+        actual1.insert(1);
+        actual1.insert(3);
+
+        LinkedList actual2 = new LinkedList();
+        actual2.insert(5);
+        actual2.insert(9);
+        actual2.insert(4);
+
+        LinkedList actual = LinkedList.mergeLists(actual1, actual2);
+        String expected = "{3} -> {4} -> {1} -> {9} -> {5} -> NULL";
+        assertEquals(expected, actual.toString());
+    }
+
+    @Test public void testMergeEmptyList(){
+
+        LinkedList actual1 = new LinkedList();
+        actual1.insert(5);
+        actual1.insert(9);
+        actual1.insert(4);
+
+        LinkedList actual2 = new LinkedList();
+
+        LinkedList actual = LinkedList.mergeLists(actual1, actual2);
+        String expected = "{4} -> {9} -> {5} -> NULL";
+        assertEquals(expected, actual.toString());
+    }
```

Comment:
This is good too since one list is empty. I like that you are thinking of different edge cases like this.